### ISSUE TITLE
[Feature Request] Text field for Mr. and Ms. in customer section

### ISSUE THREAD

**Issue by lxwulf** [InvoicePlane/InvoicePlane#1059]
It's common to use `Mr.` and `Ms` in invoices.

**Private customer example:**

```text
Mr.
Johannes Gutenberg
Bookstreet 42
55116 Mainz (a Rhein)
```

**Business customer example:**

```text
Printing Press Gutenberg Inc.
Mr. Johannes Gutenberg
Bookstreet 45
55116 Mainz (a Rhein)
```

**At the moment:**

```text
Johannes Gutenberg
Bookstreet 42
55116 Mainz (a Rhein)
```
It would be one additional field above the normal `name` field for the `private customer` example. 
On company side, it's common to use the addressing/salutation directly before the personal name.

**Comment by nielsdrost7** [comment]
refs #1007

**Comment by nielsdrost7** [comment]
@lxwulf than you for your issue. It's a duplicate of #1003 , but I'll keep your feature request as well.
See the PR I attached. It's in `draft`, but if you want you can test if out.
It has a new field in that pull-request do your database will change just a tiny bit.
Just so you know.

**Comment by lxwulf** [comment]
Thank you for the information and sorry I missed the mentioned issue! 
I have extra set up a dev server, so I can test exactly such things. :blush:

> Sorry for asking, but how can I test this commit if its in draft? I did not saw any branch with this changes. 
> Or can I also just checkout this commit?
> :point_right: :point_left:

**Comment by nielsdrost7** [comment]
@lxwulf see Pull-Request https://github.com/InvoicePlane/InvoicePlane/pull/1101
it's regarding adding a title for a client.
- `git clone invoiceplane/invoiceplane`
- `git checkout feature/1059-branch`
- just install in the ususal way (`composer install, yarn install, yarn grunt`)

**Comment by nielsdrost7** [comment]
Merged, will be released in version 1.6.2
